`data` `azurerm_shared_image_version` - consider `excludeFromLatest` for `latest` version (#21822)
hashicorp/terraform-provider-azurerm: website/docs/d/shared_image_version.html.markdown

~> **Note:** You may specify `latest` to obtain the latest version or `recent` to obtain the most recently updated version.

> ~> **Note:** In 3.0, `latest` may return the image version with `exclude_from_latest` set to `true`. Starting from 4.0, `latest` does not return the image version with `exlude_from_latest` set to `true`.

Reviewer: ```suggestion
~> **Note:** In 3.0, `latest` may return an image version with `exclude_from_latest` set to `true`. Starting from 4.0 onwards `latest` will not return image versions with `exlude_from_latest` set to `true`.
```
